src/InnerCabling/InnerCablingMap.cc

```diff
@@ -64,33 +64,45 @@ void InnerCablingMap::connectModulesToGBTs(std::map<int, std::unique_ptr<PowerCh
     const int layerOrRingNumber = (isBarrel ? layerNumber : ringNumber);
     const int numELinksPerModule = inner_cabling_functions::computeNumELinksPerModule(subDetectorName, layerOrRingNumber);
 
-    const std::pair<int, double> gbtsInPowerChain = computeNumGBTsInPowerChain(numELinksPerModule, numModulesInPowerChain, isBarrel);
+    std::pair<int, double> gbtsInPowerChain = computeNumGBTsInPowerChain(numELinksPerModule, numModulesInPowerChain, isBarrel);
     const int numGBTsInPowerChain = gbtsInPowerChain.first;
     const double numModulesPerGBTExact = gbtsInPowerChain.second;
     myPowerChain->setNumGBTsInPowerChain(numGBTsInPowerChain);
-
+    const bool isSplitOverRings = myPowerChain->isSplitOverRings(); //In this case we have to compute the correct lpGBT assignment in a more hacky way
```

Comment:
Comment unclear:  does"in this case" mean that the bool isSplitOverRings will be used to decide to use the more hacky way?

Reply:
Tried to clarify it. Indeed as you say, the bool is used to decide to use the more hacky way